kernel/power/zscal.c

```diff
@@ -38,6 +38,10 @@ USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #pragma GCC optimize "O1"
 
+#if defined(__clang__)
```

Comment:
Is your intention to disable FMA for clang for both SINGLE and DOUBLE version of this algorithm?